core/api/fetch-spec/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "git+https://github.com/polypoly-eu/polyPod.git"
   },
   "scripts": {
-    "build": "rm -rf tsconfig.tsbuildinfo dist && tsc --emitDeclarationOnly && rollup -c",
+    "build": "shx rm -rf tsconfig.tsbuildinfo dist && tsc --emitDeclarationOnly && rollup -c",
```

Comment:
Hm, I kinda like it - hadn't heard about `shx`, but at least it's only a single package as opposed to `rimraf` for removing, and who knows what for copying.

Reply:
Well, rimraf is a plugin replacement for rm -rf, which is precisely what we need.

Reply:
Also, published for the last time a year ago, version 0.3.3... [Not really my first choice for this](https://www.npmjs.com/package/shx). `rimraf` is [even older](https://www.npmjs.com/search?q=%22rm%20-rf%22), but there are other replacements. If we need to introduce some dep at the top level, I'm not sure I'd go for one with a major version 0 (or for one that's 2 years old)

Reply:
Hm, `shx` seems to be reasonably widely used. What I like about using `shx` instead of replacing `rm -rf` by a singularly focused package (e.g. `rimraf`) is that it allows us to move fast: We can do basic stuff with the shell _without_ having to hard code simple operations (via Rollup, a custom script, whatever). Using `shx` we wouldn't be limiting ourselves just to have a cross platform build, we could still pretty much do the same stuff we're used to, not stopping to use Rollup for a trivial copy, or which exact package to use for removing files, which for copying, ...

Reminds me of how OpenSSL is developed: AFAIK it's developed purely against OpenBSD, and then a _separate_ team adds a portability layer. Sounds weird, but I find that quite elegant actually :) Wouldn't want the cross platform build to slow us down,  that's why I find `shx` (or something like it) so appealing.